### ISSUE TITLE
"Attributes:" Google-style docstring support for automatic help string

### ISSUE THREAD

**Issue by Conchylicultor** [lebrice/SimpleParsing#166]
Currently, `simple_parsing` force to add comments along the attributes they are defined

It would be nice if it was also supporting the google docstring style `Attributes` field: https://google.github.io/styleguide/pyguide.html#384-classes

```python
@dataclass
class Args:
  """args.

  Attributes:
    user: Username
    verbose: Display logs
  """
  user: str
  verbose: bool = False
```

**Rational:** The `simple_parsing` conventions are not supported by standard tools like:

* CiderV hover help information:

<img width="402" alt="Screenshot 2022-09-26 at 14 43 57" src="https://user-images.githubusercontent.com/9047355/192279430-237c2262-263c-4570-a4a0-0bb1b0e1251c.png">

* Sphinx API doc generator: https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html

Currently simple parsing force to document each fields twice (as `# comment` and inside the `cls.__doc__`).

**Comment by lebrice** [comment]
Great suggestion @Conchylicultor !
I'll take a look at existing parsing tools for this, would be nicer than having to write it myself.

**Comment by brentyi** [comment]
@lebrice as an FYI I've had a really good experience with [docstring_parser](https://github.com/rr-/docstring_parser)! Could possibly also be used in #156?

**Comment by lebrice** [comment]
Sure @brentyi , I'll check it out.

However, do you get the same problem when using docstrings under the fields?

**Comment by brentyi** [comment]
Hm, my understanding with the triple-quoted attribute docstrings is:

- Sphinx would work.
- In Pyright, Attributes wouldn't show up in the `Args()` docstring, but would if you hovered over `user` in `Args(user="something")` or `args.user`.

I generally prefer this style over the monolithic `Attributes:` list.

**Comment by lebrice** [comment]
Should be fixed by #179 . Let me know if you have other issues :)
